libs/full/runtime_components/include/hpx/runtime_components/new.hpp

```diff
@@ -66,7 +68,7 @@ namespace hpx {
     ///          component instance.
     ///
     template <typename Component, typename... Ts>
-    <unspecified> new_(id_type const& locality, Ts&&... vs);
+    new_(id_type const& locality, Ts&&... vs);
```

Comment:
Let's at least write it as:
```suggestion
    auto new_(id_type const& locality, Ts&&... vs);
```
Function need a return type ;-)